addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoAlbumArt.java

```diff
@@ -214,4 +215,22 @@ private ImageType getImageType(char imgType) {
 
         return it;
     }
+
+    public @NonNull String getAlbumArtMimeType() {
+        String mimeType = "";
```

Comment:
Is empty a sensible default?

Reply:
What else it could be?

Reply:
Looking at the complete code it feels like there are two places where the mime type is determined here and based on the `imgData`. Maybe you should just return the `imageType` here so you don't need the magic return value. 

And this empty string indicates we do not do it here but should try to do it somewhere else.

Reply:
>Maybe you should just return the imageType here so you don't need the magic return value.

Actually, I implementation it already like that, but if Onkyo receiver send URL (imageType = URL), AlbumArt class download image from url and don't know the mime type for it. It that case it imageType lies as it URL rather than image.